stdlib/public/core/SIMDVector.swift

```diff
@@ -781,20 +781,35 @@ extension SIMD where Scalar: FixedWidthInteger {
 //  be replaced with @_semantics to lower directly to vector IR nodes.
 extension SIMD where Scalar: FloatingPoint {
   @_transparent
+  @differentiable(vjp: _vjpAdd(lhs:rhs:)
```

Comment:
Could you add a `// SWIFT_ENABLE_TENSORFLOW` in the beginning of every diff blob? We use this to determine whether something is a S4TF-specific change.

Reply:
Done!